augur.sh/docs/hardhat-tasks.md

```diff
@@ -70,3 +70,17 @@ use these you will need a rundown api key from rapidapi.
 ```
 yarn hardhat fetch-rundown-event --event [eventId] --key [rundownApiKey]
 ```
+
+## Fund link
+This task funds an address with 1 LINK. Call this task with a private key and network. Make sure that private key account has LINK. 
```

Comment:
An optional enhancement: check the sender's LINK balance first and give a detailed error if they don't have enough.